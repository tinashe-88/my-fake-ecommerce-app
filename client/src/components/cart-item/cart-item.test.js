import React from 'react'
import { shallow } from 'enzyme'
import CartItem from './cart-item'

it('Should render CartItem component', () => {
  const mockItem = {
    imageUrl: 'www.images.com',
    price: 1873,
    name: 'sneakers',
    quantity: 3
  }

  expect(shallow(<CartItem item={mockItem}/>)).toMatchSnapshot()
})