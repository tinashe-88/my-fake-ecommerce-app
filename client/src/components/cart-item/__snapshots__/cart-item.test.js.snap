// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render CartItem component 1`] = `
<styled.div>
  <styled.img
    alt="item"
    src="www.images.com"
  />
  <styled.div>
    <styled.span>
      sneakers
    </styled.span>
    <span
      className="price"
    >
      3
       x 
      1873
    </span>
  </styled.div>
</styled.div>
`;
